### PR TITLE
[ADD] l10n_lv: Latvian localization

### PR DESCRIPTION
**Pull request by antoine162**
Based on https://github.com/odoo/odoo/pull/112159

### CODE REVIEW

**Comment by arnis-putnins** [comment]
Thank you @antoine162 for changes you made. Yes, we agree with these changes and are very happy the merging process is continuing.

**Comment by antoine162** [comment]
Hi @arnis-putnins just a little update on where we are at :smile: 
Our Product Owner @chklop reviewed the PR, he had a few comments which I addressed in the latest push on Tuesday, and he is reviewing it again now. After that, if there still is anything that needs improving functionally, I will do those changes. And then it will go to technical review.

Thank you so much again for all this work, and I will keep you updated.

**Comment by arnis-putnins** [comment]
Hi @antoine162 Thank you so much for this work and care, we are really excited to see l10n_lv in Odoo main repository in near future.

**Comment by antoine162** [comment]
Hi @arnis-putnins just a quick update - the functional testing was good, we are waiting to get a technical reviewer at the moment (which might take a few days still) :smile:

**Comment by arnis-putnins** [comment]
@antoine162 +1 smile

File: addons/l10n_lv/data/account.account.template.csv [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+"a2230","Annual plantings","2230","asset_current","l10n_lv.chart_template_latvia",,"False"
+"a2310","Accounts receivable – trade","2310","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2318","Settlement of doubtful debtors","2318","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2320","Bills of exchange received for deliveries and customers","2320","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2330","Debts of subsidiaries","2330","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2340","Debts of related companies","2340","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2350","Settlements with other debtors","2350","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2360","Settlement of unpaid amounts of the company's capital","2360","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2370","Short-term loans for company shareholder and management","2370","asset_receivable","l10n_lv.chart_template_latvia",,"True"
+"a2380","Settlement of claims on staff","2380","asset_receivable","l10n_lv.chart_template_latvia",,"True"
```
Comment (william-andre) [inline addons/l10n_lv/data/account.account.template.csv:57]: Are really all of these receivables?
I would expect the "Settlement" accounts to be of another type :thinking:

File: addons/l10n_lv/static/description/icon.png
Comment (william-andre) [inline addons/l10n_lv/static/description/icon.png:1]: Who made this?
We can still see the flag of another country behind it...

File: addons/l10n_lv/data/res_bank_data_lv.xml
Comment (william-andre) [inline addons/l10n_lv/data/res_bank_data_lv.xml:1]: I've never seen this anywhere (ok for LT...), is this really something that we want?

Comment (antoine162) [reply addons/l10n_lv/data/res_bank_data_lv.xml:1]: I will ask one of the POs.

File: addons/l10n_lv/static/description/icon.png
Comment (antoine162) [reply addons/l10n_lv/static/description/icon.png:1]: In the end, I should remove it bc the icon will be generated automatically.

File: addons/l10n_lv/data/res_bank_data_lv.xml
Comment (antoine162) [reply addons/l10n_lv/data/res_bank_data_lv.xml:1]: Conclusion: let's remove them (online sync should add them anyway, so introducing them here might cause duplicates)

**Comment by antoine162** [comment]
Voilà, ready to merge I think
Thank you soooo much @william-andre for reviewing this :smile:

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**